minecraft_servers/plantaria/manifest.json

```diff
@@ -0,0 +1,29 @@
+{
+  "server_name": "plantaria",
+  "nice_name": "Plantaria",
+  "direct_ip": "plantaria.net",
+  "server_wildcards": [
+    "%.plantaria.net",
+    "%.plantaria.dev"
+  ],
+  "supported_languages": [
+     "de",
+     "en"
+  ],
+  "social": {
+    "web": "https://plantaria.net",
+    "web_shop": "https://store.plantaria.net",

```

Comment:
Not reachable

Reply:
the website is now removed and will only be added when it is out of maintenance.